packages/react-core/src/components/Menu/MenuGroup.tsx

```diff
@@ -8,7 +8,7 @@ export interface MenuGroupProps extends Omit<React.HTMLProps<HTMLElement>, 'labe
   /** Additional classes added to the MenuGroup */
   className?: string;
   /** Group label */
-  label?: React.ReactNode | React.FC;
+  label?: React.ReactNode;
```

Comment:
Would this count as a breaking change?

Reply:
That's a good question. I look at this as possibly breaking. The change of this type was merged 2 weeks ago and not sure if it was actually released yet. The type was changed from `string` to this. If it was released, I doubt many teams/apps have been able to update and rely on this type change and specifically pass a prop-less functional component.

Reply:
No.  It was just added this sprint.  It has not been released yet.